documentation/src/main/asciidoc/elasticsearch-integration.asciidoc

```diff
@@ -73,7 +73,7 @@ docker run -p 9200:9200 -d -v "$PWD/plugin_dir":/usr/share/elasticsearch/plugins
 ==== Dependencies in your Java application
 
 In addition to the usual dependencies like Hibernate ORM and Hibernate Search,
-you will need the new `hibernate-search-backend-elasticsearch` jar.
+you will need the new `hibernate-search-elasticsearch` jar.
```

Comment:
Actually the artifact id hasn't been changed yet. Do we have an alternative term for the notition of "Lucene Embedded vs. ES vs. Solr"? Or should we just remove "backend" and be done. Works for me, too.

Though it seems a bit we are lacking a term for this concept here (in general, not only in terms of artifact ids). "Indexing service"?